receiver/windowseventlogreceiver/metadata.yaml

```
@@ -12,3 +12,5 @@ status:
 
 tests:
   config:
+    start_at: end
+    channel: application
```

```suggestion
    channel: application

```
Nit: For the sake of GitHub's diffs, feel free to ignore.